packages/client/src/profile/Profile.ts

```diff
@@ -16,16 +19,29 @@ import {
   FollowingRequest,
   MutualFollowersProfilesQueryRequest,
   ProfileQueryRequest,
+  SetDispatcherRequest,
   SingleProfileQueryRequest,
+  TypedDataOptions,
 } from '../graphql/types.generated';
+import { execute } from '../helpers/execute';
+import {
+  getSdk as getMutationSdk,
+  Sdk as MutationSdk,
+  CreateSetDispatcherTypedDataMutation,
+} from './graphql/mutations.generated';
 import { getSdk, Sdk } from './graphql/queries.generated';
 
 export class Profile {
+  private readonly authentication: Authentication | undefined;
   private readonly sdk: Sdk;
+  private readonly mutationSdk: MutationSdk;
 
-  constructor(config: LensConfig) {
+  constructor(config: LensConfig, authentication?: Authentication) {
     const client = new GraphQLClient(config.environment.gqlEndpoint);
+
     this.sdk = getSdk(client);
+    this.mutationSdk = getMutationSdk(client);
```

Comment:
I don't know if it makes sense to separate queries from mutations, wdyt?

Reply:
What are the benefits of separating it? Adds one more step to rename the same-named imports (And also messes up auto import 😂 )

Reply:
ok, I will keep one sdk then :)